go.mod

```diff
@@ -1,20 +1,22 @@
 module sigs.k8s.io/custom-metrics-apiserver
 
-go 1.20
+go 1.21
+
+toolchain go1.21.5
```

Comment:
Do we need to set the toolchain?

Reply:
Thanks for review, this is automatically generated when I use go mod tidy, let me check it.

Reply:
```
make test
```
will fail during the building process

Reply:
Yes, have some breaking change on https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/261c3dd161e995a5bffb0e095c8d58e47edc2686/pkg/cmd/options/options.go#L102 , so i make this to draft.